### PR TITLE
[DEV-3771] Fixes Live Banner

### PR DESCRIPTION
**Pull request by richarddubay**
### What does this PR do, or why is it needed?
This PR does 2 things:
1. It updates the live banner so that when it's tapped it now goes to NewSpring's instance of Church Online.
2. It takes advantage of new configuration options so that the live banner will only display when NewSpring is actually live. We actually use a Church Online API endpoint to determine when this is. https://live.newspring.cc/api/v1/events/current

### How do I test this PR?
Open the sim.
If we are currently supposed to be live (see the API endpoint above) then the live banner should show.
If you tap on the live banner it should take you to NewSpring's instance of Church Online.
If you're testing this during a time when we are not live, you can temporarily change the API endpoint to be the one that Apollos Church currently uses (https://apollos.churchonline.org/api/v1/) so that the banner will show in order to test that the link goes where it says it should.

*Note*: This does use version `0.8.7-alpha.1` of `apollos-data-connector-church-online`. Obviously, this is an alpha version that will need to be set to the real `0.8.7` version when it is released.

---
> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] Closes [DEV-3771](https://newspring.atlassian.net/browse/DEV-3771)
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android if applicable
- [x] Set two relevant reviewers

## REVIEW

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -96,11 +96,12 @@
...
-"@apollosproject/data-connector-church-online@^0.8.4":
-  version "0.8.4"
-  resolved "https://registry.yarnpkg.com/@apollosproject/data-connector-church-online/-/data-connector-church-online-0.8.4.tgz#bf4584c89b952d66631cddf6efafb0e1198f1921"
+"@apollosproject/data-connector-church-online@^0.8.7-alpha.1":
+  version "0.8.7-alpha.1"
+  resolved "https://registry.yarnpkg.com/@apollosproject/data-connector-church-online/-/data-connector-church-online-0.8.7-alpha.1.tgz#63ee0a7c0cbc3f1be2a0eca6abe202fabaf7eb60"
+  integrity sha512-SgiKekX+26yXnQ7EIyvy6Fau2NwWEFrvaWf+X1ulomRmdRxdvAWeBUpA4Qv9QiQJT/YIqvzTpKJohLodPgAwoQ==
   dependencies:
-    "@apollosproject/data-schema" "^0.8.4"
+    "@apollosproject/data-schema" "^0.8.5"
```
Comment (IsaacHardy) [inline yarn.lock:104]: Will this cause us issue since we are still using 0.8.4 in app?

Comment (richarddubay) [reply yarn.lock:104]: @IsaacHardy Vincent says we're "totally fine". 👍

**Review by cdanwards — APPROVED** [review]
LGTM!
